sdk/appconfiguration/app-configuration/test/public/secretReference.spec.ts

```diff
@@ -12,16 +16,28 @@ import {
   secretReferenceContentType,
   SecretReferenceValue,
 } from "../../src";
-import { Recorder } from "@azure-tools/test-recorder";
 import { Context } from "mocha";
+import { Recorder } from "@azure-tools/test-recorder-new";
 
 describe("AppConfigurationClient - SecretReference", () => {
   let client: AppConfigurationClient;
   let recorder: Recorder;
-
-  beforeEach(function (this: Context) {
-    recorder = startRecorder(this);
-    client = createAppConfigurationClientForTests() || this.skip();
+  let key: string;
+  let randomString: string;
+
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    await recorder.start(recorderStartOptions);
+    client =
+      createAppConfigurationClientForTests(recorder.configureClientOptionsCoreV1({})) ||
+      this.skip();
+    recorder.variable("key-1", `key-1-${getRandomNumber()}`);
+    recorder.variable(
+      "random-string-1",
+      `random-string-1-${Math.ceil(Math.random() * 1000 + 1000)}`
```

Comment:
Is there something semantically different about this way of constructing a random and the `getRandomNumber` function?